src/zeit/cms/content/metadata.py

```diff
@@ -206,3 +208,25 @@ def set_default_channel_to_ressort(context, event):
     if channel in context.channels:
         return
     context.channels = context.channels + (channel,)
+
+
+@grok.subscribe(
+    zeit.content.article.interfaces.IArticle,
+    zope.lifecycleevent.IObjectModifiedEvent)
+def log_access_change(context, event):
+    log = zope.component.getUtility(zeit.objectlog.interfaces.IObjectLog)
+    import zeit.cms.content.sources as src
+    for description in event.descriptions:
+        if 'access' in description.attributes:
+            access_old = zeit.cms.interfaces.ICMSContent(
+                context.uniqueId).access
+            access_old_translation = src.ACCESS_SOURCE.factory.getTitle(
```

Comment:
Das habe ich gerade mal versucht anzuwenden, laufe aber in einen {AttributeError: 'FactoredContextualSource' object has no attribute 'getTitle'}